server/channels/db/migrations/mysql/000133_add_channel_banner_fields.up.sql 

```diff
@@ -0,0 +1,14 @@
+SET @preparedStatement = (SELECT IF(
+    NOT EXISTS(
+        SELECT 1 FROM INFORMATION_SCHEMA.COLUMNS
+        WHERE table_name = 'Channels'
+        AND table_schema = DATABASE()
+        AND column_name = 'BannerInfo'
+    ),
+    'ALTER TABLE Channels ADD COLUMN BannerInfo json;',
```

Comment:
I think we should add default here, considering mysql -> postgres transition both schemas should have similar structure to avoid confusion and issues. (This will also led to unnecessary diff in the comparison)

I think we should just do a test to measure how long would it take to run this on a large database. Since channels is not a very busy table, I'd say it's okay to have a small amount of locking here. Thoughts @agnivade?

Reply:
Oh good point on the schema conversion. Let's remove the default then @harshilsharma63. Apologies for the double work. I missed this part.

And no, no amount of locking is acceptable, however small it is. We need to uphold our promises :)

Reply:
Okay, so to be on the same page - I'll remove default and not null from postgres and then both postgres and mysql will have a nullable column with no default. Right?

Reply:
Yes, both of them should be `NULL` by default.

Reply:
Done.

Reply:
Yep, thanks all.